dbt_project.yml

```diff
@@ -10,12 +10,19 @@ models:
       +materialized: table
     staging:
       +schema: stg_intercom
-      +materialized: table
+      article_history:
+        +materialized: view
+      collection_history:
+        +materialized: view
+      help_center_history:
+        +materialized: view
```

Comment:
All the staging should be views, but I know going from a table to view is breaking, so I made just these new tables views. I can remove this if that makes more sense.